src/components/MainView/main-view.jsx

```diff
@@ -23,6 +29,7 @@ class MainView extends React.Component {
    *       patient in context.
    */
   componentDidMount() {
+    this.props.setLoader(true);
```

Comment:
Can we make the setLoader method name more descriptive ? 
I see that we are updating the setLoadingStatus using this method. Can we directly use setLoadingStatus here?